gateway/src/test/java/io/zeebe/gateway/api/UnavailableBrokersTest.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.0. You may not use this file
+ * except in compliance with the Zeebe Community License 1.0.
+ */
+package io.zeebe.gateway.api;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatCode;
+
+import io.atomix.cluster.AtomixCluster;
+import io.grpc.Status.Code;
+import io.zeebe.client.ZeebeClient;
+import io.zeebe.client.api.ZeebeFuture;
+import io.zeebe.client.api.command.ClientStatusException;
+import io.zeebe.client.api.command.FinalCommandStep;
+import io.zeebe.gateway.Gateway;
+import io.zeebe.gateway.impl.configuration.GatewayCfg;
+import io.zeebe.gateway.impl.configuration.NetworkCfg;
+import io.zeebe.test.util.asserts.grpc.ClientStatusExceptionAssert;
+import io.zeebe.test.util.socket.SocketUtil;
+import io.zeebe.util.sched.ActorScheduler;
+import java.io.IOException;
+import java.net.InetAddress;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Stream;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.parallel.Execution;
+import org.junit.jupiter.api.parallel.ExecutionMode;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+
+@Execution(ExecutionMode.CONCURRENT)
```

Comment:
What does this?

Reply:
Tests are executed concurrently instead of one after the other. Should be used only when we are sure that they don't interfere between each other :slightly_smiling_face:

Reply:
thanks :)